nifi-external/nifi-kafka-connect/nifi-kafka-connector/src/main/java/org/apache/nifi/kafka/connect/StatelessNiFiSinkTask.java

```diff
@@ -66,33 +64,36 @@ public String version() {
 
     @Override
     public void start(final Map<String, String> properties) {
-        logger.info("Starting Sink Task with properties {}", StatelessKafkaConnectorUtil.getLoggableProperties(properties));
+        logger.info("Starting Sink Task");
+        StatelessNiFiSinkConfig config = createConfig(properties);
 
-        final String timeout = properties.getOrDefault(StatelessKafkaConnectorUtil.DATAFLOW_TIMEOUT, StatelessKafkaConnectorUtil.DEFAULT_DATAFLOW_TIMEOUT);
+        final String timeout = config.getDataflowTimeout();
         timeoutMillis = (long) FormatUtils.getPreciseTimeDuration(timeout, TimeUnit.MILLISECONDS);
 
-        dataflowName = properties.get(StatelessKafkaConnectorUtil.DATAFLOW_NAME);
+        dataflowName = config.getDataflowName();
 
-        final String regex = properties.get(StatelessNiFiSinkConnector.HEADERS_AS_ATTRIBUTES_REGEX);
+        final String regex = config.getHeadersAsAttributesRegex();
         headerNameRegex = regex == null ? null : Pattern.compile(regex);
-        headerNamePrefix = properties.getOrDefault(StatelessNiFiSinkConnector.HEADER_ATTRIBUTE_NAME_PREFIX, "");
+        headerNamePrefix = config.getHeaderAttributeNamePrefix();
+        if (headerNamePrefix == null) {
+            headerNamePrefix = "";
+        }
```

Comment:
Instead of checking for `null` and using an empty string here, should that check be implemented in `getHeaderAttributeNamePrefix()`?